pkg/localkube/localkube.go

```diff
@@ -97,6 +100,26 @@ func (lk LocalkubeServer) GetHostIP() (net.IP, error) {
 	return utilnet.ChooseBindAddress(net.ParseIP("0.0.0.0"))
 }
 
+func (lk LocalkubeServer) getExtraConfigForComponent(component string) []util.ExtraOption {
+	e := []util.ExtraOption{}
+	for _, c := range lk.ExtraConfig {
+		if c.Component == component {
+			e = append(e, c)
+		}
+	}
+	return e
+}
+
+func (lk LocalkubeServer) SetExtraConfigForComponent(component string, config interface{}) {
+	extra := lk.getExtraConfigForComponent(component)
+	for _, e := range extra {
+		glog.Infof("Setting %s to %s on %s.\n", e.Key, e.Value, component)
+		if err := util.FindAndSet(e.Key, config, e.Value); err != nil {
+			glog.Warningf("Unable to set %s to %s. Error: %s", e.Key, e.Value, err)
```

Comment:
Not sure if we want to warn or err here.  I do see why its only a warning, however the warning is kind of hard to find.  I needed to `minikube logs | grep localkube.go` to find it.  The localkube logs aren't shown on minikube start even with full verbosity. 

Could see this being sneaky error for unsupported types (net.IP, slices, etc.)